plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java

```diff
@@ -185,15 +185,23 @@ public List<SchemaTableName> listTables(ConnectorSession session, Optional<Strin
         BigQueryClient client = bigQueryClientFactory.create(session);
 
         log.debug("listTables(session=%s, schemaName=%s)", session, schemaName);
-        String projectId = client.getProjectId();
+        String projectId;
```

Comment:
projectId is always the same between if and else, right?

Reply:
It doesn't have to be